recipes-bsp/u-boot/u-boot-internal.inc

```diff
@@ -13,5 +13,5 @@ SRC_URI += " \
     file://0002-Add-compiler-gcc6.h-header-file.patch \
     "
 SRC_URI += "file://${MACHINE}.env"
-SRC_URI += "file://target_env/*.env"
```

Comment:
I don't understand exactly why this is needed and the git log is not thta explicit.

Reply:
Because we need to have the directory copied over by yocto, not just the files in it. It's a change in Yocto Morty/Pyro which makes the build crash if we don't copy the entire directory now

Reply:
That sounds strange. So now having files in SRC_URI doesn't work anymore? Only directories?

Reply:
The WORKDIR of u-boot needs to have a directory called "target_env" in which to have the .env files. The previous SRC_URI, which I modified in this commit, only listed the .env files in the target_env. No idea how that worked, but with Yocto Pyro, if we want to have the target_env directory in the work dir, we need to list it in SRC_URI as the change does